Fix simulation scripts and scarab_stats library

5surim opened the pull request.

5surim commented:
@FlyingFish800 This is ready for review.

FlyingFish800 reviewed (dismissed):
On line 359 of slurm_runner.py, there is a typo where scarab_mode should be sim_mode. After that fix everything works

5surim commented:
@FlyingFish800 Thanks for the review and catching the typo!
I just fixed the typo and added some more color options to scarab_stats.
Could you approve this again?

FlyingFish800 reviewed (approved):
LGTM!
